app/javascript/components/admin/AuditModal.js

```diff
@@ -127,125 +127,83 @@ class AuditModal extends React.Component {
     const change = data.value;
     switch (change.name) {
       case 'locked_at':
-        // Generic audit message in case before & after values not provided
-        if (!change.details || !Array.isArray(change.details) || !change.details.length) {
-          return (
-            <span>
-              <b>Account Status</b>: Updated
-            </span>
-          );
-          // More detailed audit message when before & after values provided
-        } else {
-          return (
-            <span>
-              <b>Account Status</b>: {change.details[0] ? 'Unlocked' : 'Locked'}
-            </span>
-          );
-        }
+        return (
+          <span>
+            <b>Account Status</b>
+            <br></br>
```

Comment:
Can be shorthanded as `<br />`

Reply:
Same with all other instances

Reply:
Addressed: [7cd3591](https://github.com/SaraAlert/SaraAlert/pull/1222/commits/7cd3591f25723bcdd6b3a1f8a98b6c97ca73b6a6)